indico/modules/rb_new/util.py

```diff
@@ -162,8 +169,11 @@ def get_existing_rooms_occurrences(rooms, start_dt, end_dt, allow_overlapping=Fa
     return group_list(query, key=lambda obj: obj.reservation.room.id)
 
 
-def get_rooms_conflicts(rooms, start_dt, end_dt, repeat_frequency, repeat_interval):
-    room_conflicts = {}
+def get_rooms_conflicts(rooms, start_dt, end_dt, repeat_frequency, repeat_interval, blocked_rooms,
```

Comment:
I see this function has grown much, what about splitting it somehow to make it more readable?